src/lib/mina_base/user_command.ml

```diff
@@ -62,18 +62,19 @@ end
 
 module Gen = Gen_make (Signed_command)
 
-let gen_signed =
+let gen_signed ~signature_kind =
```

Comment:
This method is used only in `gen` below, so maybe makes sense to use `signature_kind:Testnet` rigth in the method, and have no `~signature_kind` parameter